packages/example/pages/_app.tsx

```diff
@@ -245,10 +263,14 @@ function RainbowKitApp({
       : currentTheme.accentColors[selectedAccentColor];
 
   const [isMounted, setIsMounted] = useState(false);
+
   useEffect(() => setIsMounted(true), []);
 
   const appContextProps: AppContextProps = { authEnabled };
 
+  const locales = router.locales!;
+  const locale = (router.locale || router.defaultLocale) as Locale;
```

Comment:
Is `locales` here being used? Conflicts with the drop-down locales list variable